.github/workflows/Java.yml

```diff
@@ -53,6 +51,9 @@ jobs:
           (cd examples/jdbc; make; make maven)
       - name: Deploy
         shell: bash
+        if: github.ref == 'refs/heads/master'
```

Comment:
What's the rationale behind no longer publishing artifacts on branch builds?

Reply:
Thanks for catching this, it's just wrong, the logic should be "only for tags", and it's wrong in a similar way in a bunch of other places.